src/Engine/UnitActionType.java

```diff
@@ -233,4 +234,35 @@ public String name()
       return "RESUPPLY";
     }
   }
+
+  /**
+   * Effectively a wait, but the unit ends up as a different unit at the end of it.
+   * Is a template action, and thus doesn't get a constant.
```

Comment:
template action?

Reply:
It needs to know what it's transforming *to*, so you can't just make a global TRANSFORM constant because that doesn't make sense.

"Template" isn't 100% technically correct, but it's close to what I mean and it was the first acceptable thing that came to mind.